packages/components/src/Tag/_docs/Tag.mdx

```diff
@@ -0,0 +1,71 @@
+import { Canvas, Controls , Meta } from "@storybook/blocks"
+import { ResourceLinks, KaioNotification, Installation } from "~storybook/components"
+import * as TagStories from "./Tag.stories"
+
+<Meta of={TagStories} />
+
+# Tag
+
+<ResourceLinks
+  sourceCode="https://github.com/cultureamp/kaizen-design-system/tree/main/packages/components/src/Tag"
+  figma="https://www.figma.com/file/ZRfnoNUXbGZv4eVWLbF4Az/%EF%B8%8F%F0%9F%96%BC%EF%B8%8F-Component-Gallery?node-id=9%3A37843&t=zlNKMVZd1i6MB4db-1"
+  designGuidelines="https://cultureamp.atlassian.net/wiki/spaces/DesignSystem/pages/3081928978/Tags"
+  className="!mb-8"
+/>
+
+<KaioNotification />
+
+<Installation
+  installCommand="yarn add @kaizen/components"
+  importStatement='import { Tag } from "@kaizen/components"'
+/>
+
+## Overview
+
+Tags help users quickly recognize important information about items that organize and categorize them. They visually label items with small amounts of information or the item's status, usually with keywords that help organize and categorize the items.
+
+<Canvas of={TagStories.Playground} />
+<Controls of={TagStories.Playground} />
+
+## API
+
+### Variants
+We have 3 different sets of combinations for various uses
+
+#### Status
+Use status tags to show the status of surveys, reports, action plans, and performance feedback.
+
+<Canvas of={TagStories.Status} />
+
+#### Validation
+Use validation tags for indicating the validation state such as the Cautionary style for cautioning users that the item has a validation warning that could be addressed.
+
+<Canvas of={TagStories.Validation} />
+
+#### Sentiments
+Use sentiment tags only for sentiment or other employee feedback data
+
+<Canvas of={TagStories.Sentiments} />
+
+### Sizes
+**Note:** Icons and Avatars are not supported in `small` sizes.
+
+<Canvas of={TagStories.Sizes} />
+
+### Avatar
```

Comment:
Perhaps worth noting that there are two ways of adding an Avatar - JSX.Element and AvatarProps.

Reply:
Oh, had no idea!